sdk/tables/azure-data-tables/src/samples/java/com/azure/data/tables/TableServiceClientCodeSnippets.java

```diff
@@ -27,10 +27,10 @@ public void createTable() {
             .buildClient();
         try {
             tableServiceClient.createTable("OfficeSupplies");
-        } catch (TableStorageException e) {
```

Comment:
Interesting that this had `TableStorageException` - Or was this sample previously incorrect?

Reply:
It was incorrect before. From what I can tell, `TableStorageException` was a type added at some point to expose error information in place of the internal exception type, but was never really used broadly, which is why I introduced `TableServiceException`.

Reply:
Seems like there were previously two `TableStorageException` and `TableServiceErrorException`, that are now resolved to the single `TableServiceException`?

Reply:
Yes, every error that comes from the service in the form of the generated `TableServiceErrorException` is now mapped to the public `TableServiceException`.